docs/cardinal/game/system/overview.mdx

```diff
@@ -121,39 +121,36 @@ package system
 // This provides an example of a system that doesn't rely on a message to update a component.
 func RegenSystem(world cardinal.WorldContext) error {
```

Comment:
I think we need to call this param `ctx` instead of `world` because the user's brain may confuse the `cardinal.World` they will inevitably use elsewhere, with this. There are a few other instances of this in this file also I'm going to avoid tagging them so as to not spam you!

Reply:
I went with "worldCtx" and I tried to use it every single time we reference cardinal.WorldContext. I went with just "world" or "w" when talking a cardinal.World.